spec/acceptance/validate_ruby_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'spec_helper_acceptance'
 
-describe 'Running ruby validation' do
+describe 'pdk validate ruby', module_command: true do
```

Comment:
Is it generally understood by contributors how the shared_examples is being included here? I needed to read rspec-core source to understand what _module_command_ does. It would be more readable to me if it used _it_behaves_like 'a module command'_ to link to the shared example spec code; but I'm fairly new to rspec and shared_example metadata might be a generally understood concept?